flutter/README.md

```diff
@@ -61,7 +61,7 @@ In order to track navigation events you have to add the
 `SentryNavigationObserver` to your `MaterialApp`, `WidgetsApp` or `CupertinoApp`.
 
 ```dart
-import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
```

Comment:
For `MaterialApp` one needs to import `import 'package:flutter/material.dart';`.

Reply:
good catch